static/src/javascripts-legacy/projects/common/views/svgs.js

```diff
@@ -145,10 +143,11 @@ define([
     };
 
     return function (name, classes, title) {
-        if (config.switches.webpack) {
+        // #wp-rjs - needed by karma
+        if (svgs[name].markup) { // webpack way
```

Comment:
Only 55 calls to the `svgs()` function in the whole app. And 44 icons. It's likely a bunch of these icons aren't used and are contributing to a larger than necessary commercial chunk (54k of commercial is SVGs). Probably worth killing and loading SVGs in the modules that need them. Maybe I'll make a separate PR.